tgui/packages/tgui/interfaces/Orbit.js

```diff
@@ -59,7 +61,7 @@ const OrbitedButton = (props, context) => {
     <Button
       color={color}
       onClick={() => act("orbit", {
-        name: thing.name,
+        name: thing.ref,
```

Comment:
Almost certain it's because this is `name` not `ref`.

Reply:
Timberpoes changed code to use refs, see related dm.

Reply:
Fixed was PR'd.